packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2273,25 +2273,40 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
  
      ## Pod anti-affinity can prevent the scheduler from placing Prometheus replicas on the same node.
      ## The default value "soft" means that the scheduler should *prefer* to not schedule two replica pods onto the same node but no guarantee is provided.
-@@ -487,6 +850,9 @@
+@@ -487,6 +850,24 @@
    enabled: true
    namespaceOverride: ""
  
++  ## Grafana's primary configuration
++  ## NOTE: values in map will be converted to ini format
++  ## ref: http://docs.grafana.org/installation/configuration/
++  ##
```

Comment:
The comment here comes straight from the Grafana Helm chart https://github.com/grafana/helm-charts/blob/f4cd46e1a3994d7616208ad0c34eba0f8b57aa74/charts/grafana/values.yaml#L413-L416.